packages/integrations/alpinejs/README.md

```diff
@@ -55,11 +55,11 @@ export default defineConfig({
 });
 ```
 
-Finally, restart the dev server.
+Once complete, be sure to restart the dev server by pressing `CTRL+C` and then typing `npm run astro dev` in the terminal window that's running your Astro project.
```

Comment:
```suggestion
Finally, restart the dev server.
```
There's actually a good reason to use "finally" here, as it lets the reader know that these are the end of the instructions! And, since we've explained restarting the dev server above, we don't have to do it again here. Most people will only reach this point if they've already tried the automated installation instructions and it hasn't worked for them.

Reply:
If that's the case shouldn't `finally,` be used on the `quick install`? Currently, it has `Then,`

These are two distinct instruction sets. There's no harm in making quick and manual have the same closing statement. We could do this for both:
> Finally, restart the dev server by typing `CTRL+C` and then `npm run astro dev` in the terminal window that was running Astro.